technical-steering-committee/CHARTER.md

```diff
@@ -0,0 +1,126 @@
+
+      2. Maintainers are Contributors who have earned the ability to modify ("commit") source code, documentation or other technical artifacts in any of the project’s repositories; and  
+           
+      3. A Contributor may become a Maintainer by a majority approval of the existing Maintainers. A Maintainer may choose to step down from their function, or be removed by a ⅔ majority approval of the other existing Maintainers.
+
+   f. Participation in the Project through becoming a Contributor and Maintainer is open to anyone so long as they abide by the terms of this Charter. 
+
+   g. The TSC may (1) establish workflow procedures for the submission, approval, and closure/archiving of projects, (2) set requirements for the promotion of Contributors to Maintainer status, as applicable, and (3) amend, adjust, refine and/or eliminate the roles of Contributors, and Maintainers, and create new roles, and publicly document any TSC roles, as it sees fit.
+
+   h. At the inception of the Project, the initial TSC Chair will be that Initial Member indicated in the STEERING.md file in the Project’s github repository as being the TSC Chair. As of the time of the First Election and annually thereafter, the TSC will elect a TSC Chair. The TSC Chair will preside over meetings of the TSC and will serve until their resignation or replacement by the TSC. The TSC Chair, or any other TSC member so designated by the TSC, will serve as the primary communication contact between the Project and OpenSearch Foundation, a directed fund of The Linux Foundation.
+
+   i. Responsibilities: The TSC will be responsible for all aspects of oversight relating to the Project, which may include:
+
+      1. coordinating the technical direction of the Project;
+
+      2. approving project or system proposals (including, but not limited to, incubation, deprecation, and changes to a sub-project’s scope);
+
+      3. organizing sub-projects and removing sub-projects;
+
+      4. creating sub-committees or working groups to focus on cross-project technical issues and requirements;
+
+      5. appointing representatives to work with other open source or open standards communities;
+
+      6. establishing community norms, workflows, issuing releases, security issue reporting policies and the Elections Procedures Document (defined below);
+
+      7. approving and implementing policies and processes for contributing (to be published in the CONTRIBUTING file) and coordinating with the series manager of the Project (as provided for in the Series Agreement, the "Series Manager") to resolve matters or concerns that may arise as set forth in Section 7 of this Charter;
+
+      8. discussions, seeking consensus, and where necessary, voting on technical matters relating to the code base that affect multiple projects; and
+
+      9. coordinating any marketing, events, or communications regarding the Project.
```

Comment:
This seems wrong. That's *definitely* not the job of the TSC. Point 9 is the job description of the community manager, and would be a significant distraction from the TSC's correct focus - on the steering the technical direction of the project.

Reply:
Thanks @rbowen.

I think this item came from the template we used and is present in other charters under LF, [here](https://openmessaging.cloud/charter/) and [here](https://caseontology.org/resources/charter.html). 

Removing that paragraph would be an amendment to the Technical Charter that was reviewed by the members before we announced the foundation, which requires a 2/3 vote of the entire TSC and approval by LF. I opened https://github.com/opensearch-project/community/issues/53, please add your comments.